backend/app/routes/web/recommendation.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=missing-function-docstring,no-else-return
+
+""" Presentable items UI endpoint """
+import datetime
+import random
+import uuid
+from typing import Literal
+
+import httpx
+from fastapi import APIRouter, Depends
+from httpx import AsyncClient
+
+from app.config import RECOMMENDER_ENDPOINT
+from app.generic.models.bad_request import BadRequest
+from app.schemas.search_request import TermsFacet
+from app.schemas.session_data import SessionData
+from app.solr.operations import get, search
+from app.utils.cookie_validators import cookie, verifier
+
+router = APIRouter()
+
+
+def _get_panel(panel_id: str) -> list[str]:
+    # IMPORTANT!!! recommender does not support services
+    if panel_id == "publication":
+        return ["publications"]
+    elif panel_id == "datasets":
```

Comment:
Here is the plural form `"datasets"`, in other cases there is the singular form, is this correct?